filebeat/tests/system/test_tcp_tls.py

```diff
@@ -204,16 +221,18 @@ def test_tcp_over_tls_mutual_auth_succeed(self):
         )
 
         filebeat = self.start_beat()
+        self.addCleanup(filebeat.kill_and_wait)
 
         self.wait_until(lambda: self.log_contains(
             "Start accepting connections"))
 
         sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
 
-        context = ssl.create_default_context(ssl.Purpose.CLIENT_AUTH)
+        context = ssl.create_default_context(purpose=ssl.Purpose.SERVER_AUTH)
```

Comment:
If I keep the `ssl.Purpose.CLIENT_AUTH` the test fails with `ssl.SSLError: Cannot create a client socket with a PROTOCOL_TLS_SERVER context (_ssl.c:801)`